From f4fd881063a062b3470baaf44a4d8f2ba0071098 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Fri, 9 Jun 2023 13:41:20 +0800
Subject: [PATCH] Revert "mm/memcg: Only perform the debug checks on
 !PREEMPT_RT"

This reverts commit debedeb4264e87b8625b7421b2f8f7688b42d192.

This commit caused podman does not work

Upstream-Status: Inappropriate [aptiv specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 init/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig b/init/Kconfig
index 6fd196cb9ef3..118071bef5c0 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -964,6 +964,7 @@ config CGROUP_FAVOR_DYNMODS
 
 config MEMCG
 	bool "Memory controller"
+	depends on !PREEMPT_RT
 	select PAGE_COUNTER
 	select EVENTFD
 	help
-- 
2.27.0

